ftw/simplelayout/testing.py

```diff
@@ -79,7 +81,7 @@ class SimplelayoutTestCase(TestCase):
     def assert_recursive_persistence(self, structure):
 
         def is_persistent(item):
-            if not isinstance(item, basestring) and not isinstance(item, int):
+            if not isinstance(item, basestring) and not isinstance(item, int):  # noqa
```

Comment:
Why the # noqa? we usually allow 120 letters per line.

Reply:
`basestring` can't be found according to flake8.

Reply:
ahh you use flake8 of a python 3 😏